src/picasso-definition/interactions/pan.js

```diff
@@ -48,12 +49,30 @@ const pan = ({ chart, actions, viewHandler }) => ({
         xAxisMax: xAxisMax - xDiff,
         yAxisMin: yAxisMin + yDiff,
         yAxisMax: yAxisMax + yDiff,
+        deltaX: e.deltaX,
+        deltaY: e.deltaY,
       };
 
       viewHandler.setDataView(dataView);
     },
     areaPanend(e) {
       e.preventDefault();
+      chartModel.command.setPanEnded();
+      const { componentSize, xAxisMin, xAxisMax, yAxisMax, yAxisMin } = this[eventName];
+
+      const xDiff = (xAxisMax - xAxisMin) * (e.deltaX / componentSize.width);
+      const yDiff = (yAxisMax - yAxisMin) * (e.deltaY / componentSize.height);
+
+      const dataView = {
+        xAxisMin: xAxisMin - xDiff,
+        xAxisMax: xAxisMax - xDiff,
+        yAxisMin: yAxisMin + yDiff,
+        yAxisMax: yAxisMax + yDiff,
+        deltaX: e.deltaX,
+        deltaY: e.deltaY,
+      };
```

Comment:
This code is duplicated in here and in areaPanmove... could be broken out into a separate function.